src/js/components/help/system-updates.js

```diff
@@ -37,7 +37,7 @@ const SystemUpdates = ({ docsVersion = '' }) => (
 
     <p>
       You can learn more about Mender&#39;s A&#47;B partition architecture{' '}
-      <a href={`https://docs.mender.io/${docsVersion}architecture/overview`} target="_blank">
+      <a href={`https://docs.mender.io/${docsVersion}overview/introduction`} target="_blank">
```

Comment:
Do not know if it is strictly required, but it looks to me a `/`is missing after `${docsVersion}`?

Reply:
There are several occurrences of this, so this comment applies to all of them.

Reply:
the `/` is supplied with the `docsVersion`, since this allows omitting the docsVersion/ supplying no version and automatically getting the link to the default docs version